src/libraries/Common/src/Interop/OSX/System.Security.Cryptography.Native.Apple/Interop.Hmac.cs

```diff
@@ -35,6 +35,17 @@ internal static partial class AppleCrypto
 
         [DllImport(Libraries.AppleCryptoNative, EntryPoint = "AppleCryptoNative_HmacCurrent")]
         private static extern int HmacCurrent(SafeHmacHandle ctx, ref byte pbOutput, int cbOutput);
+
+        [DllImport(Libraries.AppleCryptoNative, EntryPoint = "AppleCryptoNative_HmacOneShot")]
+        internal static unsafe extern int HmacOneShot(
+            PAL_HashAlgorithm algorithm,
+            byte* pKey,
+            int cbKey,
+            byte* pData,
+            int cbData,
+            byte* pOutput,
+            int cbOutput,
+            out int cbDigest);
```

Comment:
I know there is a difference between `byte*` and `ref byte` in the P/Invokes, but I don't remember what it is, and I thought that "we" were "happier" with the `ref byte` version.  (Open ended comment)

Reply:
> I thought that "we" were "happier" with the ref byte version

Hm. I kind of had the opposite conclusion. Some reasons for thinking:

1. This comment: https://github.com/dotnet/runtime/pull/33820#issuecomment-601423697
2. I can't find the PR but I seem to recall that avoiding the `ref` is preferable because I _think_ that means we can avoid a pinvoke stub here.

Reply:
That's fair.  I'm sure my fingers will keep writing things the other way (and it lets me avoid writing the word "unsafe"); but I agree with the comment of "ref byte feels like one byte" and perhaps start to come around.